src/Files.App/Contexts/DisplayPage/LayoutTypes.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Files.App.Contexts
+{
+	public enum LayoutTypes : ushort
```

Comment:
Isn't there already an enum for this?

Reply:
Not really. There are FolderLayoutModes (DetailsView/TilesView/ColumnView/GridView/Adaptive) and GridViewSizeKind (Small/Medium/Large). It is more convenient for commands to simply view or modify a single enum.